includes/class-wc-payment-gateway-wcpay.php

```diff
@@ -529,6 +574,8 @@ public function payment_fields() {
 			wp_localize_script( 'wcpay-checkout', 'wcpay_config', $this->get_payment_fields_js_config() );
 			wp_enqueue_script( 'wcpay-checkout' );
 
+			$this->maybe_localize_customer_data();
```

Comment:
This will break down when CC method is disabled so we need to add this line to payment_fields method in the Sepa gateway.

It does cause duplicate data to be injected when both methods are disabled but there's other things that are duplicated as well so we should handle this in a future refactor.

Reply:
I have modified the function and include the localization in both classes.
Thanks for pointing it out.